package/MDAnalysis/analysis/dihedrals.py

```diff
@@ -45,39 +45,74 @@
    function to calculate dihedral angles from atom positions
 
 
-Example application
--------------------
-This example will show how to calculate the phi and psi angles of adenylate
-kinase (AdK) and generate a basic Ramachandran plot. The trajectory is included
-within the test data files::
+Example applications
+--------------------
+
+General dihedral analysis
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :class:`~MDAnalysis.analysis.dihedrals.Dihedral` class is useful for calculating
+angles for many dihedrals of interest. For example, we can find the phi angles
+for residues 5-10 of adenylate kinase (AdK). The trajectory is included within
+the test data files::
 
    import MDAnalysis as mda
    from MDAnalysisTests.datafiles import GRO, XTC
    u = mda.Universe(GRO, XTC)
-   r = u.select_atoms("protein")    # selection of residues
 
-   from MDAnalysis.analysis.dihedrals import Ramachandran
+   # selection of atomgroups
+   ags = [res.phi_selection() for res in u.residues[4:9]]
+
+   from MDAnalysis.analysis.dihedrals import Dihedral
+   R = Dihedral(ags).run()
+
+The angles can then be accessed with :attr:`Dihedral.angles`.
+
+Ramachandran analysis
+~~~~~~~~~~~~~~~~~~~~~
+
+The :class:`~MDAnalysis.analysis.dihedrals.Ramachandran` class allows for the
```

Comment:
Please add references to the papers (Ramachandran and Janin) and add a section _References_.